test/relativistic/moving_particle_energy_test.py

```diff
@@ -1,31 +1,33 @@
 import pytest
 from symplyphysics.laws.relativistic.moving_particle_energy import calculate_particle_energy
-from sympy.physics.units import (speed_of_light, 
-                                 meter, 
-                                 second, 
-                                 kg, 
-                                 convert_to)
-import math
 
-def test_basic():
-    e = calculate_particle_energy(1 * kg, 0.5 * speed_of_light)
-    check = 1 * kg * convert_to(
-        speed_of_light, [meter, second]) ** 2 * \
-        math.sqrt(1 / (1 - 0.5 ** 2))
-    assert (convert_to(e, [meter, kg, second])) == check
-    
-
-def test_exception():
-    with pytest.raises(ValueError) as e:
-        res = calculate_particle_energy(1 * kg, 1.0 * speed_of_light)
-    with pytest.raises(ValueError) as e:
-        res = calculate_particle_energy(1 * kg, 1.1 * speed_of_light)
+from pytest import approx, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from sympy import Abs, sqrt
+from sympy.physics.units import speed_of_light
 
 
-# run from ... simplyphysics/symplyphysics dir
-# python -m pytest test/relativistic/moving_particle_energy_test.py        
+def test_basic():
+    e = calculate_particle_energy(Quantity(1 * units.kilogram), 
+                                  Quantity(0.5 * speed_of_light))
+    assert SI.get_dimension_system().equivalent_dims(e.dimension, units.energy)
+    e_nondim = convert_to(e, 
+                          units.kilogram * units.meter ** 2 / units.second ** 2).as_coeff_Mul()[0]
+    c = convert_to(speed_of_light, units.meter / units.second).as_coeff_Mul()[0]
+    check = c ** 2 * sqrt(1 / (1 - 0.5 ** 2))
```

Comment:
You are basically copypasting the formula from the law. What if it was incorrect? You won't detect an error with this test.
Look on the internet for some real value to put here.